docs/introduction.rst

```diff
@@ -99,11 +99,18 @@ This is achieved with the :class:`~factory.Sequence` declaration:
 
 .. code-block:: pycon
 
+    >>> # The sequence counter starts at 0 by default
     >>> UserFactory()
     <User: user0>
     >>> UserFactory()
     <User: user1>
 
+    >>> # The Sequence can be overridden, this still increments the sequence counter
```

Comment:
I would write this as `# A value can be provided for a sequence-driven field, but this still increments the sequence counter`

Reply:
![image](https://user-images.githubusercontent.com/42296566/122951936-f2a1e100-d37d-11eb-873f-7e45562014b9.png)
Is this ok?

Reply:
That's awesome, thanks!